### PR TITLE
Improve DC Power Example usability

### LINKED ISSUE #486: Improve usability of DC-Power example
# Bug Report

The DC-Power example should have an optional 'Device Reset' once complete to return the instrument back to a known state. Measurements are available to be used in TestStand, and should support the workflow of multiple measurements using the same session.

## Repro or Code Sample

Example:
image

## Expected Behavior

I'd like some documentation about when and why you would perform a device reset.

[AB#2661919](https://dev.azure.com/ni/94b22d7b-ad7b-4f5e-88f0-867910f91c94/_workitems/edit/2661919)

### PR DESCRIPTION
**Pull request by nathan-murphy**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md). _(Required)_
- [x]  Automatically post PR comments with images for G code changes? _(Recommended for small changes)_

### What does this Pull Request accomplish?

Improves usability by illustrating a device reset at the end of a measurement.

### Why should this Pull Request be merged?

When our team was working with this example as a starting point, we got tripped up by needing to reconfigure the instrument and not realizing that it wasn't being handled by MeasurementLink directly. A note like this would have helped save us some development/debugging time from the start.  

### What testing has been done?

Code added is exclusively in a diagram disable structure, but we have tested that the functionality works on real hardware. Original example continues to work as normal. 

Fixes #486

### CODE REVIEW

**Comment by nathan-murphy** [comment]
Wrong LV version was used. Will update with correct version.

**Comment by bkeryan** [comment]
@nathan-murphy @dixonjoel Should the Python example be updated too? https://github.com/ni/measurementlink-python/blob/main/examples/nidcpower_source_dc_voltage/measurement.py

**Comment by nathan-murphy** [comment]
@bkeryan If we started with the python measurement example it's likely that we would have run into the same issue, so yes, I think the python one should also be updated. As long as there's no dissent from @dixonjoel I'll open another issue on that repo.

File: Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi
Comment (dixonjoel) [inline Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi:1]: I'll wait for the updated LV version change.

**Review by dixonjoel — CHANGES_REQUESTED** [review]

File: Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi
Comment (nathan-murphy) [reply Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi:1]: The second commit on this PR has the VI saved for LabVIEW 2020.

Comment (dixonjoel) [reply Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi:1]: I was going to ask the same thing. What are the implications or are there any drawbacks if we just always reset / abort it?

Comment (nathan-murphy) [reply Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi:1]: It's not clear if Abort will reset all properties of a channel to default, so if you call different measurements with the same channel and abort between them I'm concerned that settings will bleed over from the first measurement into the second if the user doesn't explicitly set all values. 

I assume Reset takes longer than Abort, but I don't have HW available at the moment to benchmark this.

**Review by dixonjoel — APPROVED** [review]
Approved with suggestions.

File: Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi
Comment (nathan-murphy) [reply Source/Example Measurements/NI-DCPower Source DC Voltage/NI-DCPower Measurement/subVIs/NI-DCPower Specific Logic.vi:1]: Point of information:
The driver example referenced in this Measurement example uses Reset. I'll stick with Reset for consistency.

**Pull request merged**